### PR TITLE
Add Redux   to React Components 

### PR DESCRIPTION
**Pull request by ridabensalem**
In this milestone of the Bookstore project we have added the following requirements : 
- Connect the store to the main component of  React app.
- Define a state slice that will contain the books.
- Display books.
- Add a <Button> component, which includes:
- [ ] An event handler that adds a book to the books array 
- Add a <Button> component, which includes:
- [ ] An event handler that removes a book from the books array 
- Add a <Button> component, which includes:
- [ ] - An event handler that checks the status

### CODE REVIEW

File: src/redux/Books/BooksSclice.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
...
+];
 const booksSlice = createSlice({
   name: 'books',
-  initialState: {
-    list: [],
-  },
+  initialState,
   reducers: {
     addBook: (state, action) => {
       state.list.push(action.payload);
```
Comment (youmari) [inline src/redux/Books/BooksSclice.js]: - [ ] you're not exporting the  `bookSclice` as default so it returns an error when you run the app please make sure to fix that in order to review your app
Screenshot 2023-03-07 at 14 39 37

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @ridabensalem , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @ridabensalem , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ]... ✅ Green linters

## Required Changes ♻️
 

- [ ] While you have made great progress, your app still can not be run, the are some dependencies issues, so I changed the eslint version but still has the same issue, please make sure you fix this issue.



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# Pull Request Approved ✅ 🎊 🎉 
## Hello @ridabensalem 👋 

## Things to Appreciate 
- [ ] Correct Flow ✅ 
- [ ] Clean Code 👍 
- [ ] No Linter Errors 🔥 
- [ ] Descriptive Pull Request and Summary 👌 

### Your project is complete!  There is nothing else to say other than it's time to merge it.

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 ☑️ 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**